feature/experimental/p4rt/tests/p4rt_election/p4rt_election_test.go

```diff
@@ -278,7 +278,7 @@ func canWrite(t *testing.T, args *testArgs) (bool, error) {
 		}
 		return false, writeErr
 	}
-	if !*deviations.P4RTMissingDelete {
+	if !deviations.P4RTMissingDelete(ondatra.DUT(t, "dut")) {
```

Comment:
Unfortunate that the dut pointer wasn't handy here, but I agree that this is preferable to threading the dut through the call chain, when our goal is to make the deviations go away entirely. In other migrations, let's be sure to reuse the existing dut variable where we can. (Nothing to change here, just FYI.)

Reply:
Agree. I did check to see if I could reuse the dut here, but since it was not available went this route. 

Will reuse the existing dut variable whereever I can.